en/vespa8-release-notes.html

```diff
@@ -574,6 +574,18 @@ <h3 id="strict-mode">Strict mode enabled by default</h3>
   attribute in services.xml for details.
 </p>
 
+<h3 id="request-headers-for-remote-in-access-log">Request headers controlling remote host/port in access log</h3>
```

Comment:
I think our convention is to use the full header as id, but it's not a big deal for me personally.